src/inc/LibraryIncludes.h

```diff
@@ -64,6 +64,7 @@
 #ifndef BLOCK_GSL
 #define GSL_MULTI_SPAN_H
 #include <gsl/gsl>
+#include <gsl/span_ext>

```

Comment:
span_ext is where the make_span is coming from?

Reply:
Yes